examples/dem/3d-bunny-drill/bunny-drill-loading.prm

```diff
@@ -91,14 +91,14 @@ end
 #---------------------------------------------------
 
 subsection insertion info
-  set insertion method                               = volume
-  set inserted number of particles at each time step = 10000
-  set insertion frequency                            = 100000
-  set insertion box points coordinates               = 0.1, -0.07, -0.07 : 0.24, 0.07, 0.07
-  set insertion distance threshold                   = 1.10
-  set insertion maximum offset                       = 0.10
+  set insertion method                               = plane
+  set inserted number of particles at each time step = 200
+  set insertion frequency                            = 8000
+  set insertion plane point                          = 0.025, 0, 0
+  set insertion plane normal vector                  = -1, 0, 0
+  set insertion maximum offset                       = 0.00
```

Comment:
Same comment

Reply:
maximum offset default value is not 0, it's 1 for a reason I have no idea.
This should be fixed in another PR.